pkg/controllers/dynakube/extension/eec/statefulset.go

```diff
@@ -295,7 +294,7 @@ func buildContainerVolumeMounts(dk *dynakube.DynaKube) []corev1.VolumeMount {
 
 func setVolumes(dk *dynakube.DynaKube) func(o *appsv1.StatefulSet) {
 	return func(o *appsv1.StatefulSet) {
-		tlsSecretName := extensionsControllerTlsSecretName
+		tlsSecretName := dk.Name + consts.ExtensionsTlsSecretSuffix
```

Comment:
i found we use it quite often, can we create a function or method? to handle this logic easily in future?

Reply:
[Fixed](https://github.com/Dynatrace/dynatrace-operator/pull/3778/commits/c1fb51c95e19d48604f6c04fc85b977cb720dace)